docs/UsersGuide/source/RunSRW.rst

```diff
@@ -1177,11 +1174,19 @@ The regional workflow can be run using standalone shell scripts in cases where t
       export EXPTDIR=`pwd`
       setenv EXPTDIR `pwd`
 
+#. Set the ``PDY`` and ``cyc`` environment variables. ``PDY`` refers to the first 8 characters (YYYYMMDD) of the ``DATE_FIRST_CYCL`` variable defined in the ``config.yaml``. ``cyc`` refers to the last two digits of ``DATE_FIRST_CYCL`` (HH) defined in ``config.yaml``. For example, if the ``config.yaml`` file defines ``DATE_FIRST_CYCL: '2019061518'``, the user should run:
+
+   .. code-block:: console 
+      
+      export PDY=20190615 && export cyc=18 
```

Comment:
@MichaelLueken I just noticed that PR #475 changed PDY/cyc to PDYext/cycext. Should I update that here?

Reply:
@gspetro-NOAA The change from `PDY/cyc` to `PDYext/cycext` was done in `scripts/exregional_get_extrn_mdl_files.sh` to correct an issue on WCOSS2 (the original fix was to directly alter the path of PDY and cyc, but that wasn't approved, leading to the creation of these two new variables).  The rest of the scripts still require PDY and cyc and these are are still being created in the `parm/FV3LAM_wflow.xml` XML file.

I think it is fine to keep the PDY and cyc as they currently are and not update them in the `RunSRW.rst` documentation.